docs/copied-from-beats/dashboards.asciidoc

```diff
@@ -0,0 +1,83 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/dashboards.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+
+{beatname_uc} comes packaged with example Kibana dashboards, visualizations,
+and searches for visualizing {beatname_uc} data in Kibana. Before you can use
+the dashboards, you need to create the index  pattern, +{beat_default_index_prefix}-*+, and
+load the dashboards into Kibana. To do this, you can either run the `setup`
+command (as described here) or
+<<configuration-dashboards,configure dashboard loading>> in the
++{beatname_lc}.yml+ config file.
+
+NOTE: Starting with {beatname_uc} 6.0.0, the dashboards are loaded via the Kibana API.
+This requires a Kibana endpoint configuration. You should have configured the
+endpoint earlier when you
+<<{beatname_lc}-configuration,configured {beatname_uc}>>. If you didn't,
+configure it now.
+
+Make sure Kibana is running before you perform this step. If you are accessing a
+secured Kibana instance, make sure you've configured credentials as described in
+<<{beatname_lc}-configuration>>.
+
+To set up the Kibana dashboards for {beatname_uc}, use the appropriate command
+for your system.
```

Comment:
The next section is an overview over APM Server cmds, as the instruction for the Power Shell is valid for all instructions, how about adding the Powershell instruction here instead:

![screen shot 2018-01-29 at 14 34 15](https://user-images.githubusercontent.com/5555349/35512906-96aac8b0-0501-11e8-9b34-cc9e5a132762.png)

Reply:
good idea, will do it in beats in a separate PR when this is done.